### ISSUE TITLE
syntax without ||

### ISSUE THREAD

**Issue by mopoIo** [mmattDonk/AI-TTS-Donations#90]
I think it would be a good feature to make it work without having to add `||` between messages. I know nothing about regex or coding but I would make it stop when it sees a new voice `text: ` or a sound `(1)`.

I don't know if I should make another feature request for this other suggestions since they go in hand with this syntax feature but I would make it so if you make a mistake it would go back to the previous voice, for example:
`drake: hello i am drake (1) spngbob: i am still drake`
it would read it **all** with **Drake's** voice as "hello i am drake" (sound effect) "spngbob: i am still drake". 
(even the spngbob part)
This way you stop the bot from stopping completely when a user sends a wrong voice id.

I would also add a default non ai voice (or a default ai voice / a way to pick a default ai voice) to read the first segment if it's wrong since the beginning, that way you can always get tts output even if the person sending the message is a pepega

For example: if the person forgets to add a starting voice or makes a spelling mistake on the first voice id, it would read it as Brian until it sees a valid voice id.

**Comment by gabrera** [comment]
I don't know python and sorry if this is bad github etiquette but this is how I would do it: Instead of using `||` as the split point, I would use `: ` and ` (` and the last character (the `: ` has a space next to it and the ` (` has a space behind it)

**First step:**
I would start by checking if the first character it's a `(`. If it is I would set a default voice and then I would check if the text next to it matches a playsound id such as `2) `  `3) `   `4) ` etc. If it matches then I would queue that sound then I would search for the next split point. 
If it doesn't match I would set a default voice then I would search for the next split point.

If it doesn't start with ` (` I would check if the first word has a `: ` next to it, if it does I would check if that word matches a voice-list, if it does match I would save that as the first voice, if it doesn't I would use a default voice. Then I would search for the next split point.

**Split points:**
if it's `: ` i would check if the word behind it matches a voice-list, if it does I would save that voice as the next voice and I would remove the last word from the previous sentence (since the last word will be the voice id for the next sentence) and that would be the previous message (played with the previous saved voice). Then I would search for the next split point.
If it doesn't match I would use the voice I saved on the previous scan for the previous sentence, then I would search for the next split point.

if it's ` (` i would check if the text next to it matches a playsound id such as `2) `  `3) `   `4) ` etc. If it matches I would save the text behind it as a message (minus the ` (`) to play with the previous voice I saved and I would queue that sound then I would search for the next split point. 
If it doesn't then I would read the previous sentence (minus the ` (`) with the previous voice and I would search for the next split point.

if it's the last character I would stop the text reading there.

This is probably the worst way of doing it but I've been scratching my brain and this is what I could come up with
